documentation/config_file_auth.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: default
+title: "Puppet Server Configuration Files: auth.conf"
+canonical: "/puppetserver/latest/config_file_auth.html"
+---
+
+[`trapperkeeper-authorization`]: https://github.com/puppetlabs/trapperkeeper-authorization
```

Comment:
@gguillotte Okay, that makes sense.

We do cover most of the required elements in this section.  `sort-order` and `name` are the major ones that are missing.  We don't touch on the `version` setting underneath the `authorization` section either, which is required but always expected to be "1" for now.

Also, we don't explicitly touch on the `allow` or `deny` settings in this doc.  `deny` is probably more of a niche case but `allow` would definitely be worth covering - especially with respect to the extra level of detail around the forms that an `allow` entry can take on.  One of `allow`, `deny`, or `allow-unauthenticated` is required at a minimum for each rule; `allow` and `deny` can also be used together for the same rule.

I'm open to you covering these others on a separate PR if that's how you'd prefer to do it.